frontend/app/views/tier/upgrade/paidToPaid.scala.html

```diff
@@ -0,0 +1,148 @@
+@(
+    currentTier: com.gu.membership.salesforce.Tier,
+    targetTier: com.gu.membership.salesforce.Tier,
+    userFields: model.PrivateFields,
+    pageInfo: model.PageInfo,
+    paidPreviewOpt: Option[model.Zuora.PaidPreview],
+    currentSubscription: model.Zuora.SubscriptionDetails
+)(implicit token: play.filters.csrf.CSRF.Token)
+
+@import helper._
+@import org.joda.time.DateTime
+@import views.support.Prices._
+@import views.support.Dates._
+@import views.support.Asset
+
+@main("Upgrade to " + targetTier, pageInfo=pageInfo) {
+
+    <main role="main" class="page-content l-constrained">
+
+        @fragments.page.pageHeader("Great, glad to see you've decided to become a " + targetTier)
```

Comment:
This is just a personal thing so ignore the following sentence, but I *hate* this copy (it's in a few other places), specifically the "glad to see" bit. It implies a speaker ("*I'm* glad to see...") and it's not clear *who* is glad to see you're joining. I feel better now for having this moan. Thanks for coming on this journey with me. Now carry on.